package.json

```diff
@@ -132,9 +133,7 @@
     "webpack-node-externals": "^1.7.2"
   },
   "resolutions": {
-    "eslint-scope": "3.7.1",
-    "//": "Avoids use of insecure version of minimist as of 2020/03/30. Revisit.",
```

Comment:
Hey, lets keep a similar comment for the new resolution you added.

Reply:
Good idea :)